### ISSUE TITLE
Formatting of long function block names with extends and implements

### ISSUE THREAD

**Issue by Roald87** [Barteling/STweep#7]
Currently the this is formatted version of a very long function name. 

```
FUNCTION_BLOCK FB_SomeSeriouslyLongFunctionBlockNameToTest EXTENDS
	FB_AnotherSeriouslyLongFunctionBlockNameToTest IMPLEMENTS
	I_SomeVeryLongInterfaceNameThatShoudlBeEvenLongerToBreak,
	I_AnotherVeryLongInterFaceNameJustToTestItAgain
```

I think it would be better to have the `EXTENDS` and `IMPLEMENTS` in front of the names:

```
// Option 1
FUNCTION_BLOCK FB_SomeSeriouslyLongFunctionBlockNameToTest 
EXTENDS
	FB_AnotherSeriouslyLongFunctionBlockNameToTest 
IMPLEMENTS
	I_SomeVeryLongInterfaceNameThatShoudlBeEvenLongerToBreak,
	I_AnotherVeryLongInterFaceNameJustToTestItAgain

// Option 2
FUNCTION_BLOCK FB_SomeSeriouslyLongFunctionBlockNameToTest 
EXTENDS FB_AnotherSeriouslyLongFunctionBlockNameToTest 
IMPLEMENTS I_SomeVeryLongInterfaceNameThatShoudlBeEvenLongerToBreak,
    I_AnotherVeryLongInterFaceNameJustToTestItAgain

// Option 3
FUNCTION_BLOCK FB_SomeSeriouslyLongFunctionBlockNameToTest 
    EXTENDS FB_AnotherSeriouslyLongFunctionBlockNameToTest 
    IMPLEMENTS I_SomeVeryLongInterfaceNameThatShoudlBeEvenLongerToBreak,
       I_AnotherVeryLongInterFaceNameJustToTestItAgain
```

My preference would be option 1. And then with a single interface you could also do:

```
FUNCTION_BLOCK FB_SomeSeriouslyLongFunctionBlockNameToTest 
EXTENDS FB_AnotherSeriouslyLongFunctionBlockNameToTest 
IMPLEMENTS I_SomeVeryLongInterfaceNameThatShoudlBeEvenLongerToBreak
```

Or something else.

- STweep v 1.0.5-beta
- [settings.txt](https://github.com/Barteling/STweep/files/5803271/settings.txt)

**Comment by Barteling** [comment]
Agree, there is room for improvement here

**Comment by Barteling** [comment]
I fixed this in 1.0.6-Beta for TwinCAT, just released. And for Codesys 1.4.4-Beta (not yet released).

This is your option 3 more or less i think, which was now the easiest to implement.

future improvement might be needed here, for example the optional indent of extends and implements
Formatting result with line break at 100 and keep existing line breaks turned of.

<pre>FUNCTION_BLOCK FB_SomeSeriouslyLongFunctionBlockNameToTest
    EXTENDS FB_AnotherSeriouslyLongFunctionBlockNameToTest
    IMPLEMENTS I_SomeVeryLongInterfaceNameThatShoudlBeEvenLongerToBreak,
        I_AnotherVeryLongInterFaceNameJustToTestItAgain</pre>
